src/platform/windows/publish.cpp

```diff
@@ -29,7 +31,7 @@ using namespace std::literals;
 #define SV(quote) __SV(quote)
 
 extern "C" {
-#ifndef __MINGW32__
+#ifndef _WIN32
```

Comment:
Similar to case in PolicyConfig. If we don't need them with `_WIN32` (either MinGW or MSVC), you can just remove them.